var/spack/repos/builtin/packages/py-viridian/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyViridian(PythonPackage):
+    """Virus assembler from amplicon sequencing reads."""
+
+    homepage = "https://github.com/iqbal-lab-org/viridian"
```

Comment:
This repo has been renamed to cylon, but the cylon and viridian names have already been taken on PyPI, so we need to call it something else. It also says not to use this package, use viridian-workflow instead. Do we need this package?

Reply:
the cylon package on pypi is hosted on https://github.com/RocknRoot/Cylon.
The project is archived since Jan 14, 2021 and deprecated by its author.